packages/lesswrong/lib/collections/votes/views.ts

```diff
@@ -29,3 +35,20 @@ Votes.addView("tagVotes", function () {
   }
 })
 ensureIndex(Votes, {collectionName: 1, votedAt: 1})
+
+Votes.addView("userPostVotes", function ({userId, voteType, collectionName, after, before}) {
+  return {
+    selector: {
+      collectionName: collectionName,
+      userId: userId,
```

Comment:
This is a security vulnerability that deanonymizes votes

Reply:
Does this tank the entire concept? 

I'm not familiar enough with this part of the architecture to have a clear sense of the right way to do this.

I see we have a "currentUserVote" applied via makeVoteable. If I want to get all my votes from a given time period (without knowing in advance which posts are connected to those votes), what's the best way to do that?